sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Repository/ContainerRepository.cs

```diff
@@ -174,7 +179,7 @@ public virtual async Task<Response> DeleteAsync(CancellationToken cancellationTo
             }
         }
 
-        /// <summary> Delete the repository. </summary>
+        /// <summary> Delete the repository and artifacts that share its name. </summary>
```

Comment:
nit: Is the following also correct? If so, I think this more explicit wording may be a bit more clear.
```suggestion
        /// <summary> Delete the repository and all artifacts that share the name of this repository. </summary>
```